Bloggist/src/aura/PostCreation/PostCreationController.js

```diff
@@ -0,0 +1,31 @@
+({
+	clickCreatePost : function(component, event, helper) {
+		var validPost = component.find('postform').reduce(function (valid, inputCmp){
+            inputCmp.showHelpMessageIfInvalid();
+            return valid && inputCmp.get('v.validity').valid;
```

Comment:
If this component is not a top-class one, you should not worry about the validity of the author's name. Also, it's not a good idea to relate entities by a name. If anything, you should have an "author" attribute and pass on a valid contact so you can relate the entities.

Reply:
Thought that this was used in any component to check for weird possible inputs. It will be deleted.